apps/charge-accounts-service/src/common/config/configuration.ts

```diff
@@ -9,15 +9,18 @@ export default () => ({
         entrypointAddress: process.env.ENTRYPOINT_PRODUCTION_CONTRACT_ADDRESS_V_0_1_0,
         url: process.env.RPC_URL || 'https://rpc.fuse.io',
         etherspotWalletFactoryContractAddress:
-          process.env.ETHERSPOT_WALLET_FACTORY_PRODUCTION_CONTRACT_ADDRESS_V_0_1_0
+          process.env.ETHERSPOT_WALLET_FACTORY_PRODUCTION_CONTRACT_ADDRESS_V_0_1_0,
+        usdcContractAddress: process.env.USDC_CONTRACT_ADDRESS_MAINNET
```

Comment:
Placing USDC’s address under the paymaster configuration is confusing, it will be clearer to separate token-specific configurations into their own dedicated section. This avoids mixing paymaster-specific parameters with token contract addresses.

Reply:
@NikolayRn separated